iroh-net/src/relay/client.rs

```diff
@@ -987,13 +981,12 @@ impl Actor {
         }
     }
 
-    async fn recv_detail(&mut self) -> Result<(ReceivedMessage, usize), ClientError> {
-        if let Some((_client, client_receiver)) = self.relay_client.as_mut() {
+    async fn recv_detail(&mut self) -> Result<ReceivedMessage, ClientError> {
+        if let Some((_client, client_receiver)) = self.relay_conn.as_mut() {
```

Comment:
```suggestion
        if let Some((_conn, client_receiver)) = self.relay_conn.as_mut() {
```

maybe? not sure about the receiver part